### PR TITLE
Optionally disable Barbican and Manila

### PR DESCRIPTION
**Pull request by xek**
Fix the Makefile and standalone.sh to forward
the appropriate env variables for disabling
Barbican and Manila.

Jira: https://issues.redhat.com/browse/OSPRH-6857

### CODE REVIEW

File: devsetup/Makefile [hunk 1]
```diff
@@ -60,6 +60,9 @@ EDPM_COMPUTE_ADDITIONAL_HOST_ROUTES ?= '[]'
 TLS_ENABLED ?= false
 DNS_DOMAIN ?= localdomain
 
+BARBICAN_SERVICE_ENABLED ?= false
+MANILA_SERVICE_ENABLED ?= false
+
```
Comment (gouthampacha) [inline devsetup/Makefile:65]: right now; both these services are enabled on the standalone deployment; your change will break that: https://github.com/openstack-k8s-operators/install_yamls/blob/ff5075d356345f9e7e945ac53a164aebae055c52/devsetup/standalone/openstack.sh#L23-L24

Please default to True and disable these locally if you wish.

Comment (xek) [reply devsetup/Makefile:65]: Thanks, I made the change

File: devsetup/Makefile [hunk 2]
```diff
@@ -60,6 +60,9 @@ EDPM_COMPUTE_ADDITIONAL_HOST_ROUTES ?= '[]'
 TLS_ENABLED ?= false
 DNS_DOMAIN ?= localdomain
 
+BARBICAN_SERVICE_ENABLED ?= false
```
Comment (marios) [inline devsetup/Makefile:63]: here we have default to false

in standalone.sh we have default to true (line 51/52):

```
BARBICAN_ENABLED=${BARBICAN_ENABLED:-true}
MANILA_ENABLED=${MANILA_ENABLED:-true}
```


can we be consistent on this

File: devsetup/scripts/standalone.sh [hunk 3]
```diff
@@ -138,6 +138,8 @@ export COMPUTE_DRIVER=${COMPUTE_DRIVER:-"libvirt"}
 export IP=${IP}
 export GATEWAY=${GATEWAY}
 export STANDALONE_VM=${STANDALONE_VM}
+export BARBICAN_ENABLED=${BARBICAN_ENABLED}
```
[the remark this answers is not in the document]
Comment (xek) [reply devsetup/scripts/standalone.sh:141]: yep, I'll just add the export keyword on line 51-52

File: devsetup/Makefile (hunk 2 above)
Comment (xek) [reply devsetup/Makefile:63]: Right, that's an error, it should default to true, as to not change the previous behavior

File: devsetup/scripts/standalone.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (xek) [reply devsetup/scripts/standalone.sh]: I added the keywords for all other duplicated variable forwarding/initialization

[the remark this answers is not in the document]
Comment (xek) [reply devsetup/scripts/standalone.sh]: Sorry, when I did that, I didn't notice these exports are placed in a different script, the $CMDS_FILE, so all of these should stay in $CMDS_FILE as is. So it looks to me no change is required on this comment?

**Review by marios — APPROVED** [review]

**Pull request merged**